src/HomeScreen/hooks/useAppFirstTime.js

```diff
@@ -0,0 +1,20 @@
+import { useEffect, useState } from "react";
+import AsyncStorage from "@react-native-async-storage/async-storage";
+
+const useAppFirstTime = () => {
+  const [isFirstTime, setIsFirstTime] = useState(true);
+  useEffect(() => {
+    (async () => {
+      const value = await AsyncStorage.getItem("hasOpenedAppBefore");
```

Comment:
The constant name `value` is too generic. I think you should name it like `openedAppBefore` or `openedAppBeforeValue` etc